### PR TITLE
toString

### LINKED ISSUE #1: add toString method
should return something like `"y(x) = 4.34x - 12"`

### PR DESCRIPTION
**Pull request by maasencioh**
Unifies toString method for the different regressions

close #1

### CODE REVIEW

**Comment by targos** [comment]
Why all the Istanbul ignore?

**Comment by maasencioh** [comment]
in base-regression it's because in the other classes are implemented the methods, and the load function it's pretty much the same for all the regressions

**Comment by targos** [comment]
IMO it is not a good reason to disable the coverage analysis. It should
only be done if it is not possible or too difficult to add tests. It is not
bad to have a coverage <100%. Means there is room for improvement.

File: test/non-linear-regression.js [hunk 1]
```diff
@@ -15,8 +15,8 @@ describe('Non-linear regression', function () {
             result.quality.r2.should.greaterThan(0.8);
             result.quality.chi2.should.lessThan(0.1);
             result.quality.rmsd.should.lessThan(0.01);
-            result.toString(4).should.equal('y = 4.907*x^2');
-            result.toLaTeX(4).should.equal('y = 4.907x^{2}');
+            result.toString(4).should.equal('f(x) = 4.907 * x ^ 2');
```
Comment (targos) [inline test/non-linear-regression.js]: Maybe the exponent should stay as x^2 ?
/cc @lpatiny

Comment (lpatiny) [inline test/non-linear-regression.js]: I would also leave as x^2

**Comment by maasencioh** [comment]
@targos @lpatiny now the exponents are like x^2 and there's test cases for the base-regression, I still have the doubt if the `toLaTeX` should have the same rules? I change it to have f(x), but I would like that you see the test cases for that and tell me what you think how is it now

**Comment by lpatiny** [comment]
For the the way it looks now is really good
image

**Comment by maasencioh** [comment]
@lpatiny but those are not the ones with the latest changes, but indeed it shouldnt change the rendering

**Comment by andcastillo** [comment]
There is something weird with the rendering. If I write one of the equations that have fractions in latex, I get this:
codecogseqn
But in the
screen shot 2016-09-27 at 9 16 19 am
 Luc's comment, It looks like the value over and the fraction line is bold.

**Comment by maasencioh** [comment]
then this is what you expect?

@mljs/collaborators

**Comment by maasencioh** [comment]
ping @mljs/collaborators

**Comment by lpatiny** [comment]
I wonder if when you write a f(x) the 'f' should not be a little bit more fancy.

If I look in wikipedia: https://en.wikipedia.org/wiki/Function_(mathematics)

I see something like

image

**Comment by maasencioh** [comment]
I found this package in LaTeX, but I'm still not convinced in force the user to use an extra library https://www.ctan.org/pkg/mathrsfs

**Comment by lpatiny** [comment]
I think actually that it will be rendered correctly based on some example

image

I think you should merge the code

**Pull request merged**